packages/mock/tests/mocking-compatibility.spec.ts

```diff
@@ -484,33 +507,28 @@ describe('Mock retro-compatibility', () => {
     });
   });
 
-  // FIXME
-  test.skip('can support explicit UnionType mock', () => {
```

Comment:
This test was targeting pre-v8 behaviour, so I've changed it over to a simple hardcoding: https://www.graphql-tools.com/docs/mocking/#breaking-change-mock-functions-signature

Sorry if it's extra noise in the PR, I can revert if preferred.

Reply:
It's fine! I just wanted to make sure if it is directly related :)